murakami/exporters/http.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+import json
+
+import requests
+
+from murakami.exporter import MurakamiExporter
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPExporter(MurakamiExporter):
+    """This exporter sends data to an HTTP endpoint."""
+    def __init__(
+            self,
+            name="",
+            location=None,
+            network_type=None,
+            connection_type=None,
+            config=None,
+    ):
+        super().__init__(
+            name=name,
+            location=location,
+            network_type=network_type,
+            connection_type=connection_type,
+            config=config,
+        )
+        logging.debug(config)
+        self._url = config.get("url")
+
+    def push(self, test_name="", data=None, timestamp=None):
+        # Make a JSON payload with the expected format
+        data = json.loads(data)
+        json_data = {
+            'apiVersion': 1,
+            'data': data
+        }
+
+        # POST the payload.
+        resp = requests.post(self._url, data=json.dumps(json_data))
```

Comment:
This should be `resp = requests.post(self._url, json=json_data)`, or else the content-type header won't be set correctly to `content/json` and the data won't be parsed.